modules/standard/FileSystem.chpl

```diff
@@ -173,6 +173,54 @@ proc exists(name: string): bool {
   return ret;
 }
 
+/* Returns the user id associated with the file or directory specified by
+   name.  Returns an errors that occurred via an out parameter.
```

Comment:
Typo: Returns _any_ errors... This typo is copied to all four new header comments.